eng/clipack/Common.projitems

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+
+    <ArchiveName>aspire-cli-$(CliRuntime)</ArchiveName>
+    <ArchiveFormat Condition="$(CliRuntime.StartsWith('linux-'))">tar.gz</ArchiveFormat>
+    <ArchiveFormat Condition="!$(CliRuntime.StartsWith('linux-'))">zip</ArchiveFormat>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Archives" />
+  </ItemGroup>
+
+  <Target Name="PublishToDisk">
```

Comment:
NIT: don't change this now, but might be good to include in your next PR a small comment here saying that the Microsoft.DotNet.Build.Tasks.Archives targets will automatically invoke this target when the project is published so that others in the future understand how this is plugged in.